main.tf

```diff
@@ -137,6 +137,8 @@ module "cloudwatch_log" {
 
   retention_in_days = var.retention_in_days
 
+  permissions_boundary = var.permissions_boundary
```

Comment:
What do you think of prefixing the inlut with `logging_` so it's flear the perm boundary is for the cloudwatch logging module's resources ?

```suggestion
  permissions_boundary = var.logging_permissions_boundary
```